torch/csrc/distributed/copy_utils.h

```diff
@@ -0,0 +1,55 @@
+#ifndef THDP_COPY_UTILS_H
+#define THDP_COPY_UTILS_H
+
+extern THDTensorDescriptor* THDPModule_makeDescriptor(PyObject *obj);
+template <typename TensorSrc>
+void THDPInsertCopyFunctionFromWorker(
+  THPCopyList& copyList,
+  void (*copyFunc)(THDTensorDescriptor* x, TensorSrc *z))
+{
+  auto wrapper = [copyFunc](PyObject* dst_, PyObject* src_) {
+    TensorSrc* src = THPTypeInfo<TensorSrc>::cdata(src_);
+
+    PyThreadState *_save = NULL;
```

Comment:
Is this variable modified by some macros or Python internal code? Because it looks like it is not used.